scripts/release/mule/deploy/deb/deploy.sh

```diff
@@ -2,27 +2,15 @@
 
 set -e
 
-if [ -z "$STAGING" ]
-then
-    echo "[$0] Staging is a required parameter."
-    exit 1
-fi
-
-if [ -z "$CHANNEL" ]
+if [ -z "$NETWORK" ]
 then
-    echo "[$0] Channel is a required parameter."
+    echo "[$0] Network is a required parameter."
     exit 1
 fi
 
-if [[ ! "$CHANNEL" =~ ^beta$|^stable$ ]]
-then
-    echo "[$0] Repository values must be either \`beta\` or \`stable\`."
-    exit 1
-fi
-
-if [ -z "$VERSION" ]
+if [ -z "$STAGING" ]
 then
-    echo "[$0] Version is a required parameter."
+    echo "[$0] Staging is a required parameter."
```

Comment:
We're not sure what the allowed values should be.  For us, it will most likely be the same, but one requirement is to not have any of our private buckets hardcoded or otherwise alluded to in code.

Reply:
ok